clustered/server/src/main/java/org/ehcache/clustered/server/EhcacheActiveEntity.java

```diff
@@ -105,6 +105,8 @@
 class EhcacheActiveEntity implements ActiveServerEntity<EhcacheEntityMessage, EhcacheEntityResponse> {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(EhcacheActiveEntity.class);
+  static final String SYNC_DATA_SIZE_PROP = "sync.data.size.threshold";
+  private static final long SYNC_DATA_SIZE_THRESHOLD = 64 * 1024;
```

Comment:
Better prefix that constant with `DEFAULT_`, and I wonder if 64 KB is a reasonable default. I was thinking of something bigger, like 1 MB. Were there perf test results indicating that 64 KB ought to be enough?